drv/BinaryOperator.drv

```diff
@@ -60,6 +60,10 @@ public interface KEY_BINARY_OPERATOR KEY_GENERIC extends BinaryOperator<KEY_GENE
 	@Override
 	@SuppressWarnings("boxing")
 	default KEY_CLASS apply(final KEY_CLASS x, final KEY_CLASS y) {
+#if !defined JDK_PRIMITIVE_BINARY_OPERATOR || KEY_WIDENED
```

Comment:
I do not understand this modification. The only method of the functional interface is apply() with primitive types. The method you are delegating to delegates to that method.

Reply:
I was fixing the following:
```
public interface IntBinaryOperator extends BinaryOperator<Integer>, java.util.function.IntBinaryOperator {
	  default Integer apply(final Integer x, final Integer y) {
// it was a recursive apply(Integer, Integer) call here before the change
// as apply(int, int) is not generated for IntBinaryOperator
	 return applyAsInt(x.intValue(), y.intValue()); 
	}
}
```

Reply:
I see. No, it's my code that's wrong, in the sense that I did not realize JDK's operators have that funny name. I would like to follow the same structure as in other classes as Function, so there will be an apply() method and applyAsInt() will delegate.